src/deno/stable/functions/readFile.ts

```diff
@@ -8,7 +8,7 @@ export const readFile: typeof Deno.readFile = async function readFile(
   { signal } = {},
 ) {
   try {
-    return await nodeReadFile(path, { signal });
+    return new Uint8Array(await nodeReadFile(path, { signal }));
```

Comment:
To avoid copying, we might want:

```JavaScript
const buf = await nodeReadFile(path, { signal });
return new Uint8Array(buf.buffer, buf.offset, buf.length);
```

Reply:
Thanks! But I used `buf.byteOffset` for the 2nd because the type checker seems unhappy with `buf.offset` and I couldn't find it in node.js docs (though it's available on the runtime..)

Reply:
`buf.offset` is available because nodejs Buffer extends the Uint8Array interface.